src/perf_client/perf_client.h

```diff
@@ -115,6 +115,8 @@ namespace client
         ->required(true)
         ->check(CLI::ExistingFile);
       app.add_option("--ca", ca_file)->required(true)->check(CLI::ExistingFile);
+      app.add_option("--jwt", jwt)
```

Comment:
Since this is setting the Bearer header, and isn't jwt-specific, can we make this --bearer or --bearer-token?